StyleCop.Analyzers/StyleCop.Analyzers/SpacingRules/SA1000KeywordsMustBeSpacedCorrectly.cs

```diff
@@ -115,13 +115,22 @@ private static void HandleSyntaxTree(SyntaxTreeAnalysisContext context)
 
                 case SyntaxKind.CheckedKeyword:
                 case SyntaxKind.UncheckedKeyword:
-                    if (token.GetNextToken().IsKind(SyntaxKind.OpenBraceToken))
+                    switch (token.Parent.Kind())
                     {
+                    case SyntaxKind.CheckedStatement:
+                    case SyntaxKind.UncheckedStatement:
+                    case SyntaxKind.OperatorDeclaration:
```

Comment:
❔ Does this need to support conversion operators as well?

Reply:
It does. Thank you! I didn't realize it was a different syntax kind. On its way.

Reply:
Updated